sql/version_20171107_upgrade.sql

```diff
@@ -1,6 +1,7 @@
 --
 -- New content size records 255 -> 1024
 --
+USE `namedmanager`;
```

Comment:
Shouldn't need to have this in here, it will break compatibility for anyone who's DB isn't called "namedmanager". The schema migration tool should be setting the correct DB when the changes are applied.

Reply:
> Shouldn't need to have this in here, it will break compatibility for anyone who's DB isn't called "namedmanager". The schema migration tool should be setting the correct DB when the changes are applied.

Ok. I heard you, corrected.